yml/imsim.yaml

```diff
@@ -8275,3 +8275,51 @@ tables:
     datatype: long
     mysql:datatype: BIGINT
     description:
+- name: match_ref
+  '@id': '#match_ref'
+  description: Match indices and metadata for a reference catalog matched to an object table
```

Comment:
This description sounds kinda abstract. E.g. you reference "a reference catalog" instead of the very specific `truthSummary` and `object` tables that it contains matches for.